js/tools/eraser.js

```diff
@@ -0,0 +1,33 @@
+/* global app */
+window.tools.eraser = {
+  name: 'eraser',
+  icon: '/images/eraser.png',
```

Comment:
in order for this image to load u need to have a file called `eraser.png` inside of the project's `images` folder, i don't see one in ur repo's [image's folder](https://github.com/yruann/paintArtware2.1/tree/main/images) so i'm guessing either u never created one, or u might have forgotten to *commit* the addition of that file before pushing ur code.